Application Settings/DependentList_Test.cls

```diff
@@ -22,55 +22,66 @@
  
 @isTest
 private class DependentList_Test {
-     public static testmethod void testCountry() {
-         List<Country__c> countryList = new List<Country__c>();
-         Country__c countryInstance1 = new Country__c(Name= 'India');
-         Country__c countryInstance2 = new Country__c(Name= 'USA');
-         Country__c countryInstance3 = new Country__c(Name= 'Pakistan');
-         countryList.add(countryInstance1);
-         countryList.add(countryInstance2);
-         countryList.add(countryInstance3);
+    public static testMethod void testGetAllCountry() {
         
-         Test.startTest();
-         insert countryList;
-         Test.stopTest();
+        // Create test data for country
+        List<Country__c> countryList = new List<Country__c>();
+        Country__c countryInstance1 = new Country__c(Name = 'India');
+        Country__c countryInstance2 = new Country__c(Name = 'USA');
+        Country__c countryInstance3 = new Country__c(Name = 'Pakistan');
+        countryList.add(countryInstance1);
+        countryList.add(countryInstance2);
+        countryList.add(countryInstance3);
         
-         DependentListController DependentListInstance = 
-            new DependentListController();
-         List<SelectOption> optionList = new List<SelectOption>();
-         optionList = DependentListInstance.selectedCountry();
-         System.assertEquals(3,optionList.size()); 
+        // Inserting country data
+        insert countryList;
+        
+        
+        
+        // New excution context
+        Test.startTest();
+		DependentListController DependentListInstance = new DependentListController();
```

Comment:
There should not be any indentation. The line should start exactly below Test.startTest();